srml/support/src/dispatch.rs

```diff
@@ -601,18 +593,25 @@ macro_rules! decl_module {
 			$( $deposit_event )*
 		}
 
-		$(
-			decl_module! {
-				@impl_function
-				$mod_type<$trait_instance: $trait_name>;
-				$origin_type;
-				$from;
-				$(#[doc = $doc_attr])*
-				$fn_vis fn $fn_name (
-					$from $(, $param_name : $param )*
-				) $( -> $result )* { $( $impl )* }
-			}
-		)*
+		/// Public dispatchable functions.
```

Comment:
Is correct, but all dispatchable functions/calls are "public". The ones that will be visible in the documentation are somewhat special, as being public functions.

Reply:
should I write "exposed dispatchable functions" here ? or just remove the line

Reply:
I removed the line